website/docs/d/container_app.html.markdown

```diff
@@ -283,6 +285,18 @@ A `custom_domain` block supports the following:
 
 ---
 
+A `ip_security_restriction` block supports the following:
+
+* `action` - The IP-filter action. `Allow` or `Deny`.
+
+* `description` - Describe the IP restriction rule that is being sent to the container-app.
+
+* `ip_address_range` - CIDR notation to match incoming IP address.
+
+* `name` - Name for the IP restriction rule.
```

Comment:
We need to change the language slightly because it's a data source. We don't need to provide information on valid values or defaults, just information on what the property is.
```suggestion
* `action` - The IP-filter action.

* `description` - Description of the IP restriction rule that is being sent to the container-app.

* `ip_address_range` - CIDR notation that matches the incoming IP address.

* `name` - Name for the IP restriction rule.
```

Reply:
Fixed.